lib/mixins/property-effects.js

```diff
@@ -926,7 +926,7 @@ function setupCompoundStorage(node, binding) {
       // We may also want to consider doing this for `textContent` and
       // `innerHTML`.
       if (target === 'className') {
-        node = wrap(node);
+        node = wrapIfNeeded(node);
```

Comment:
Seems like setting `className` on an otherwise uninteresting element in a SR (i.e. didn't need to have inside/outside patched) needs to always be wrapped so we don't lose scoping.

Might explain the test failure here? https://travis-ci.org/Polymer/polymer/builds/580988656#L780 (although unclear why this would only fail on Chrome 41?)

Reply:
Fixed. The issue is that `patch` needs to be the fallback to wrapping so that this works when instance patching is required.